cmd/ocm-backplane/login/login_test.go

```diff
@@ -168,5 +170,25 @@ var _ = Describe("Login command", func() {
 
 			Expect(err).To(BeNil())
 		})
+
+		It("Check KUBECONFIG when login into multi clusters.", func() {
+			args.manager = false
+			args.multiCluster = true
+			_ = utils.ModifyTempKubeConfigFileName(trueClusterId)
```

Comment:
Why was the error not handled here intentionally?

Reply:
Handle the error